trellis/tools/trellis-cli/README.md

```diff
@@ -0,0 +1,19 @@
+# Trellis CLI
+
+## Usage
+Usage information can be obtained using:
+```
+./trellis-cli -h
```

Comment:
There is no `-h`. You just run the binary without any command to get the help output.

Also, let's just use `bazel run //trellis/tools/trellis-cli` as the way to invoke the binary